### DISCUSSION TITLE
GraphQL.org blog post draft

### DISCUSSION THREAD

**Discussion by bignimbus** [graphql/composite-schemas-wg#157] [General]
Hi all, I've started a [short rough draft](https://docs.google.com/document/d/1zpUxG4dRP61z-bMyHhRb2MHdDtTeJPo9iCE8WjUQeTg/edit) for a graphql.org blog past based on what we discussed in last week's WG.  It's intentionally brief as a conversation starter - if there are details to add or change, please make edits/suggestions with a brief note for visibility.  And also feel free share the link with your orgs.  I figured we'd transfer over to markdown after we've all taken a pass at the content.

**Comment by benjie** [comment]
Thanks for spearheading this, Jeff! I've made a few small suggestions.

**Comment by smyrick** [comment]
LGTM!

**Comment by bignimbus** [comment]
Posted some edits this morning, the substance is that the first paragraph focuses less on GraphQL itself in the first paragraph and just talks about this specific pattern.  "GraphQL Federation" is invoked early as there is a lot of cross-vendor use of that term at the moment.  LMK what you think, please feel free to suggest edits!

**Comment by dsandip** [comment]
Hey @bignimbus, left a suggestion/comment in the post.

**Comment by bignimbus** [comment]
@benjie suggested an alternative draft that I think is a better starting point.  Replaced the prior text with his workup!
